src/_posts/databases/postgresql/2000-01-01-troubleshooting.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Troubleshooting Scalingo for PostgreSQL
+nav: Troubleshooting
+modified_at: 2023-12-01 00:00:00
+tags: databases postgresql addon
+index: 9
+---
+
+## Identifying Performances Issues
+
+
+## Understanding Idle Queries
+
+PostgreSQL® tends to use all the available memory if there is enough indices to
+fill the memory. If there is too many indices to fit into the memory, some of
+them are stored on the disk. In this situation, queries needing these indices
+will be slowed down. PostgreSQL® first needs to load the indices from the disk
+into the RAM which takes some time. The memory usage on the "Metrics" tab of
```

Comment:
I understand the point of having a paragraph describing Postgresql's memory consumption and I agree with the idea of explaining that it's a good thing that Postgresql uses all the memory available, but I don't agree with the explanation given, I think it's over simplified (it's not just about indices). With the help of a very well known generative AI I'd rather propose something like this:  
```suggestion
PostgreSQL optimizes database performance by effectively using all available memory. It stores frequently accessed data in memory to reduce the need for disk reads and writes, speeding up query processing. This approach not only enhances the system's efficiency but also requires careful memory management to prevent the system from using swap memory. Excessive usage of swap memory can significantly slow down database operations, so PostgreSQL is designed to maximize its memory usage without resorting to swap, ensuring both high performance and system stability.
```

Reply:
Well, for my defense, that's just a copy-paste from our current documentation :yum: